Makefile

```diff
@@ -76,19 +73,20 @@ docs: ## generate Sphinx HTML documentation, including API docs
 
 # Define PIP_COMPILE_OPTS=-v to get more information during make upgrade.
 PIP_COMPILE = pip-compile --upgrade $(PIP_COMPILE_OPTS)
```

Comment:
We've started also using the `--rebuild` option; from the latest revision to OEP-18:

> By default pip-compile uses a cache of calculated dependency relationships to improve the performance of subsequent runs. Unfortunately, the results of this cache are sometimes used even after a new package release has changed the set of packages it depends on. To avoid generating incorrect requirements files due to this, it’s best to always use the --rebuild option when running pip-compile.